common/src/main/java/tachyon/Constants.java

```diff
@@ -137,7 +137,10 @@
   public static final String MASTER_KEYTAB_KEY = "tachyon.master.keytab.file";
   public static final String MASTER_PRINCIPAL_KEY = "tachyon.master.principal";
   public static final String MASTER_RETRY_COUNT = "tachyon.master.retry";
-
+  public static final String MASTER_BALANCER_STRATEGY_TYPE = "tachyon.master.balance.strategy";
+  public static final String MASTER_BALANCER_STRATEGY_CLASS = 
```

Comment:
Since we have `MASTER_BALANCER_STRATEGY_CLASS` now, the above configuration option is redundant.